test/viewmodels/user_centauri_points_view_model_unit_test.dart

```diff
@@ -0,0 +1,63 @@
+import "package:fake_cloud_firestore/fake_cloud_firestore.dart";
+import "package:flutter_test/flutter_test.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/models/database/user/user_firestore.dart";
+import "package:proxima/services/database/firestore_service.dart";
+import "package:proxima/services/database/user_repository_service.dart";
+import "package:proxima/viewmodels/user_centauri_points_view_model.dart";
+
+import "../mocks/data/firestore_user.dart";
+
+void main() {
+  group("User Centauri points ViewModel unit testing", () {
+    late FakeFirebaseFirestore fakeFireStore;
+    late UserRepositoryService userRepo;
+
+    late ProviderContainer container;
+
+    late List<UserFirestore> availableUsers;
+
+    setUp(() async {
+      fakeFireStore = FakeFirebaseFirestore();
+
+      userRepo = UserRepositoryService(
+        firestore: fakeFireStore,
+      );
+
+      availableUsers = FirestoreUserGenerator.generateUserFirestore(10);
+      await setUsersFirestore(fakeFireStore, availableUsers);
+
+      //Prepare the list of providers to override.
+      container = ProviderContainer(
+        overrides: [
+          firestoreProvider.overrideWithValue(fakeFireStore),
+          userRepositoryServiceProvider.overrideWithValue(userRepo),
+        ],
+      );
+    });
```

Comment:
I think `userRepo` isn't use in the tests. 
Thus I believe we can remove it.

```suggestion
    late FakeFirebaseFirestore fakeFireStore;

    late ProviderContainer container;

    late List<UserFirestore> availableUsers;

    setUp(() async {
      fakeFireStore = FakeFirebaseFirestore();

      availableUsers = FirestoreUserGenerator.generateUserFirestore(10);
      await setUsersFirestore(fakeFireStore, availableUsers);

      //Prepare the list of providers to override.
      container = ProviderContainer(
        overrides: [
          firestoreProvider.overrideWithValue(fakeFireStore),
        ],
      );
    });
```

Reply:
You are completely right !

Removed in [7126752](https://github.com/ProximaEPFL/proxima/pull/348/commits/7126752cd987a85b2aa4aeac237f6501bbe51336)

Thank you !